sormas-backend/src/main/java/de/symeda/sormas/backend/event/eventimport/EventImportFacadeEjb.java

```diff
@@ -0,0 +1,615 @@
+			// If the string is smaller than the length of the expected date format, throw an exception
+			if (entry.length() < 10) {
+				throw new ImportErrorException(
+					I18nProperties.getValidationError(Validations.importInvalidDate, buildEntityProperty(entryHeaderPath)));
+			} else {
+				pd.getWriteMethod()
+					.invoke(element, DateHelper.parseDateWithException(entry, userService.getCurrentUser().getLanguage().getDateFormat()));
+				return true;
+			}
+		}
+		if (propertyType.isAssignableFrom(Integer.class)) {
+			pd.getWriteMethod().invoke(element, Integer.parseInt(entry));
+			return true;
+		}
+		if (propertyType.isAssignableFrom(Double.class)) {
+			pd.getWriteMethod().invoke(element, Double.parseDouble(entry));
+			return true;
+		}
+		if (propertyType.isAssignableFrom(Float.class)) {
+			pd.getWriteMethod().invoke(element, Float.parseFloat(entry));
+			return true;
+		}
+		if (propertyType.isAssignableFrom(Boolean.class) || propertyType.isAssignableFrom(boolean.class)) {
+			pd.getWriteMethod().invoke(element, Boolean.parseBoolean(entry));
+			return true;
+		}
+		if (propertyType.isAssignableFrom(AreaReferenceDto.class)) {
+			List<AreaReferenceDto> areas = areaFacade.getByName(entry, false);
+			if (areas.isEmpty()) {
+				throw new ImportErrorException(
+					I18nProperties.getValidationError(Validations.importEntryDoesNotExist, entry, buildEntityProperty(entryHeaderPath)));
+			} else if (areas.size() > 1) {
+				throw new ImportErrorException(
+					I18nProperties.getValidationError(Validations.importAreaNotUnique, entry, buildEntityProperty(entryHeaderPath)));
+			} else {
+				pd.getWriteMethod().invoke(element, areas.get(0));
+				return true;
+			}
+		}
+		if (propertyType.isAssignableFrom(RegionReferenceDto.class)) {
+			List<RegionReferenceDto> region = regionFacade.getByName(entry, false);
+			if (region.isEmpty()) {
+				throw new ImportErrorException(
+					I18nProperties.getValidationError(Validations.importEntryDoesNotExist, entry, buildEntityProperty(entryHeaderPath)));
+			} else if (region.size() > 1) {
+				throw new ImportErrorException(
+					I18nProperties.getValidationError(Validations.importRegionNotUnique, entry, buildEntityProperty(entryHeaderPath)));
+			} else {
+				pd.getWriteMethod().invoke(element, region.get(0));
+				return true;
+			}
+		}
+		if (propertyType.isAssignableFrom(UserReferenceDto.class)) {
+			UserDto user = userFacade.getByUserName(entry);
+			if (user != null) {
+				pd.getWriteMethod().invoke(element, user.toReference());
+				return true;
+			} else {
+				throw new ImportErrorException(
+					I18nProperties.getValidationError(Validations.importEntryDoesNotExist, entry, buildEntityProperty(entryHeaderPath)));
+			}
+		}
+		if (propertyType.isAssignableFrom(String.class)) {
+			pd.getWriteMethod().invoke(element, entry);
+			return true;
+		}
+
+		return false;
+	}
+
+	private boolean isPersonSimilarToExisting(PersonDto referencePerson) {
```

Comment:
Move this to sormas-backend.util.DtoHelper (also refactor CaseImportFacadeEjb accordingly)